arcaflow_plugin_stressng/stressng_schema.py

```diff
@@ -203,318 +149,296 @@ class StressNGParams:
-                "by the stressor and kernel time."
-            ),
-        }
-    )
-    bogo_ops_per_second_real_time: float = dataclasses.field(
-        metadata={
-            "id": "bogo-ops-per-second-real-time",
-            "name": "Bogus operations per second in real time",
-            "description": (
-                "real time measurement is how long the run took based "
-                "on the wall clock time "
-                "(that is, the time the stressor took to run)."
-            ),
-        }
-    )
-    wall_clock_time: float = dataclasses.field(
-        metadata={
-            "id": "wall-clock-time",
-            "name": "Wall Clock Time",
-            "description": "The time the stressor took to run",
-        }
-    )
-    user_time: float = dataclasses.field(
-        metadata={
-            "id": "user-time",
-            "name": "CPU User Time",
-            "description": "The CPU time spent in user space",
-        }
-    )
-    system_time: float = dataclasses.field(
-        metadata={
-            "id": "system-time",
-            "name": "CPU System Time",
-            "description": "The CPU time spent in kernel space",
-        }
-    )
-    cpu_usage_per_instance: float = dataclasses.field(
-        metadata={
-            "id": "cpu-usage-per-instance",
-            "name": "CPU usage per instance",
-            "description": (
-                "is the amount of CPU " "used by each stressor instance"
-            ),
-        }
-    )
+    stressor: typing.Annotated[
+        str,
+        schema.name("Stressor"),
+        schema.description("Type of stressor for workload"),
+    ]
+
+    max_rss: typing.Annotated[
+        int,
+        schema.id("max-rss"),
+        schema.name("Max RSS"),
+        schema.description("Maximum resident set size"),
+    ]
+
+    bogo_ops: typing.Annotated[
+        int,
+        schema.id("bogo-ops"),
+        schema.name("Bogus Operations"),
+        schema.description("Number of stressor loop iterations"),
+    ]
+
+    bogo_ops_per_second_usr_sys_time: typing.Annotated[
+        float,
+        schema.id("bogo-ops-per-second-usr-sys-time"),
+        schema.name("Bogus operations per second per user and sys time"),
+        schema.description(
+            "is the bogo-ops rate divided by the user + system time."
+            "This is the real per CPU throughput "
+            "taking into consideration "
+            "all the CPUs used and all the time consumed "
+            "by the stressor and kernel time."
+        ),
+    ]
+
+    bogo_ops_per_second_real_time: typing.Annotated[
+        float,
+        schema.id("bogo-ops-per-second-real-time"),
+        schema.name("Bogus operations per second in real time"),
+        schema.description(
+            "real time measurement is how long the run took based "
+            "on the wall clock time "
+            "(that is, the time the stressor took to run)."
+        ),
+    ]
```

Comment:
I suspect that this description is incorrect.  I think it should be something more like,
> The average number of stressor loop iterations completed per second of wall-clock time

or, just,
> The number of operations performed per wall-clock second